apps/financial-aid/web-veita/src/routes/MySettings/MySettings.tsx

```diff
@@ -0,0 +1,182 @@
+import React, { useContext, useState } from 'react'
+
+import {
+  ApplicationOverviewSkeleton,
+  LoadingContainer,
+} from '@island.is/financial-aid-web/veita/src/components'
+import {
+  Text,
+  Box,
+  Input,
+  ToggleSwitchCheckbox,
+} from '@island.is/island-ui/core'
+import { AdminContext } from '@island.is/financial-aid-web/veita/src/components/AdminProvider/AdminProvider'
+
+import * as styles from '@island.is/financial-aid-web/veita/src/components/Profile/Profile.css'
```

Comment:
We are following the rule that styles are used for the styles with this component and the general styles are named after them. Following that rule this style should be name `profileStyles`